### ISSUE TITLE
max_topic_alias means no Topic Aliases supported, not without limit

### ISSUE THREAD

**Issue by ckrey** [emqx/emqx#1835]
#### Environment

- OS: Debian9
- Erlang/OTP: 19
- EMQ: 3.0 beta 2

#### Description

emqx.conf:
```
## Maximum Topic Alias, 0 means no limit.
##
## Value: 0-65535
mqtt.max_topic_alias = 0
```

The 0 is transmitted in the CONNACK:
```
19:51:33.391 [debug] MQTT(10.0.2.2:60465): RECV <<16,24,0,4,77,81,84,84,5,2,0,60,3,34,0,10,0,8,77,81,84,84,118,53,84,65>>
19:51:33.391 [debug] MQTT(@10.0.2.2:60465): RECV CONNECT(Q0, R0, D0, ClientId=MQTTv5TA, ProtoName=MQTT, ProtoVsn=5, CleanStart=true, KeepAlive=60, Username=undefined, Password=undefined)
19:51:33.392 [debug] MQTT(MQTTv5TA@10.0.2.2:60465): SEND CONNACK(Q0, R0, D0, AckFlags=0, ReasonCode=0)
19:51:33.392 [debug] MQTT(10.0.2.2:60465): SEND <<32,21,0,0,18,39,0,16,0,0,36,2,37,1,42,1,41,1,34,0,3,40,1>>
19:51:33.393 [debug] MQTT(10.0.2.2:60465): Keepalive at the interval of 45
19:51:34.326 [debug] MQTT(10.0.2.2:60465): RECV <<130,16,0,2,0,0,10,77,81,84,84,118,53,84,65,47,35,0>>
19:51:34.326 [debug] MQTT(MQTTv5TA@10.0.2.2:60465): RECV SUBSCRIBE(Q1, R0, D0, PacketId=2, TopicFilters=[{<<"MQTTv5TA/#">>,
                           #{nl => 0,qos => 0,rap => 0,rc => 0,rh => 0}}])
19:51:34.327 [debug] MQTT(MQTTv5TA@10.0.2.2:60465): SEND SUBACK(Q0, R0, D0, PacketId=2, ReasonCodes=[0])
19:51:34.327 [debug] MQTT(10.0.2.2:60465): SEND <<144,4,0,2,0,0>>
19:51:35.574 [debug] MQTT(10.0.2.2:60465): RECV <<224,0>>
19:51:35.574 [debug] MQTT(MQTTv5TA@10.0.2.2:60465): RECV DISCONNECT(Q0, R0, D0, ReasonCode=0)
19:51:35.574 [debug] MQTT(10.0.2.2:60465): Terminated for normal
19:51:35.574 [info] MQTT(MQTTv5TA@10.0.2.2:60465): Shutdown for normal
```

But the spec says something different:

<img width="1134" alt="bildschirmfoto 2018-09-18 um 20 52 37" src="https://user-images.githubusercontent.com/4027289/45709786-22947800-bb85-11e8-80b6-3e7b0e221edc.png">

**Comment by Gilbert-Wong** [comment]
Yes , you are right, the comment description is wrong, but there is no problem in code.

**Comment by Gilbert-Wong** [comment]
Sorry, I checked that the validation for topic alias is not implemented in CONNECT packet. @tigercl would add these validation.

**Comment by Gilbert-Wong** [comment]
The topic alias has been fully implemented in emqx 3.0 beta4, but we forgot to modify the description in emqx.conf. The description would be fixed in next edition of emqx.
